components/dropdown/dropdown-content-mixin.js

```diff
@@ -1064,15 +1149,6 @@ export const DropdownContentMixin = superclass => class extends LocalizeCoreElem
 	}
 
 	_renderContent() {
-		const positionStyle = {};
```

Comment:
This was moved below and adjusted to be more dynamic, since we may be applying more than just the left/right position.